partiql-logical-planner/src/call_defs.rs

```diff
@@ -39,10 +44,9 @@ impl CallDef {
                 }
             }
 
-            return (overload.output)(actuals);
+            return Ok((overload.output)(actuals));
         }
-
-        todo!("mismatched formal/actual arguments to {}", &self.names[0])
```

Comment:
The line that would previously cause a panic if wrong number of function arguments provided.